test/parallel/test-runner-mock-timers.js

```diff
@@ -351,6 +357,65 @@ describe('Mock Timers Test Suite', () => {
         assert.deepStrictEqual(order, ['f1', 'f2']);
       });
     });
+
+    describe('process.nexTick Suite', () => {
+      it('should keep nextTick working if timers are disabled', (t, done) => {
+        const now = Date.now();
+        const timeout = 2;
+        const expected = () => now - timeout;
+        process.nextTick(common.mustCall(() => {
+          assert.strictEqual(now - timeout, expected());
+          done();
+        }));
+      });
+
+      it('should work with the same params as the original nextTick', (t) => {
+        t.mock.timers.enable(['nextTick']);
+        const fn = t.mock.fn();
+        const args = ['a', 'b', 'c'];
+        process.nextTick(fn, ...args);
+        t.mock.timers.tick(9999);
+
+        assert.strictEqual(fn.mock.callCount(), 1);
+        assert.deepStrictEqual(fn.mock.calls[0].arguments, args);
+      });
+
+      it('should advance in time and trigger timers when calling the .tick function', (t) => {
+        t.mock.timers.enable(['nextTick']);
+        process.nextTick(common.mustCall(1));
+        t.mock.timers.tick(0);
+      });
+
+      it('should execute in order if process.nextTick is called multiple times', (t) => {
+        t.mock.timers.enable(['nextTick']);
+        const order = [];
+        const fn1 = t.mock.fn(common.mustCall(() => order.push('f1'), 1));
+        const fn2 = t.mock.fn(common.mustCall(() => order.push('f2'), 1));
```

Comment:
Why are these mocks? can this be simplified to
```suggestion
        const fn1 = common.mustCall(() => order.push('f1'), 1);
        const fn2 = common.mustCall(() => order.push('f2'), 1);
```